spec/_attachments/ic.did

```diff
@@ -121,6 +128,16 @@ type node_metrics = record {
     num_block_failures_total : nat64;
 };
 
+type log_record = record {
+  idx: nat;
+  timestamp: nat;
```

Comment:
Inconsistent with the ICRC standards. The standards use `nat64`, see [here](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/ICRC-1.did#L2) for example.

Reply:
Fixed, switched to `nat64` and also changed the name to `timestamp_nanos` to be consistent with the other examples in this file.